src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -20,6 +20,10 @@ module sonic-portchannel {
 		revision-date 2019-07-01;
 	}
 
+	import ietf-inet-types {
+		prefix inet;
+	}
+
 	description "PORTCHANNEL yang Module for SONiC OS";
 
```

Comment:
Please add the revision to reflect the new additions

Reply:
Addressed

Reply:
Addressed